llvm-external-projects/iree-dialects/lib/Dialect/LinalgExt/Passes/ConvertConv2DToWinograd.cpp

```diff
@@ -68,6 +68,8 @@ static DenseElementsAttr foldFilterTransform(
   const int &ic = shape[2];
   const int &oc = shape[3];
   const int64_t numElements = inputTileSize * inputTileSize * ic * oc;
+  FloatType floatType = elementType.cast<FloatType>();
```

Comment:
style nit: Use `auto` because cast already spells the type.

```suggestion
  auto floatType = elementType.cast<FloatType>();
```

question: is it guaranteed that the element type is always float type? If so, can we pass `FloatType` instead of `Type`?

Reply:
Yes, it is always guaranteed it is a float type (this is asserted). So will change to pass float type instead.